library/src/blas2/rocblas_tbmv.hpp

```diff
@@ -0,0 +1,371 @@
+/* ************************************************************************
+ * Copyright 2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+#ifndef __ROCBLAS_TBMV_HPP__
+#define __ROCBLAS_TBMV_HPP__
+#include "handle.h"
+#include "rocblas.h"
+#include "utility.h"
+
+// Makes a copy of xa into ya.
+template <typename T, typename U, typename V>
+__global__ void tbmv_copy_helper(rocblas_int    n,
```

Comment:
Could consider moving this to a shared location for reuse.

Reply:
This was a straight up copy of the copy_kernel from L1 blas. I changed it to include rocblas_copy.hpp and call that directly.